mintapi/signIn.py

```diff
@@ -568,13 +645,6 @@ def get_web_driver(
         "please use login_and_get_token or sign_in",
         DeprecationWarning,
     )
-    if headless and mfa_method is None:
```

Comment:
To clarify my understanding, is this gone because now the auto-detection means omitting MFA method can lead to a successful result?

Reply:
Yes that is correct. The only time it's necessary is when Mint asks you to pick an option, which really I haven't seen come up too often.